src/pages/partners.tsx

```diff
@@ -152,88 +163,36 @@ export default function partners(): ReactElement {
 										src="/partners/versatile.png"
 									/>
 									<Cell
-										alt="Slingshot"
-										src="/partners/slingshot.png"
-									/>
-									<Cell
-										alt="Versatile"
-										src="/partners/versatile.png"
-									/>
-									<Cell
-										alt="Slingshot"
-										src="/partners/slingshot.png"
+										alt="Deloitte"
+										src="/partners/deloitte.png"
 									/>
 								</SimpleGrid>
 							</TabPanel>
 							<TabPanel>
 								<SimpleGrid
-									minChildWidth="250px"
-									spacing="40px"
-								>
-									<Cell
-										alt="Slingshot"
-										src="/partners/slingshot.png"
-									/>
-									<Cell
-										alt="Versatile"
-										src="/partners/versatile.png"
-									/>
-									<Cell
-										alt="Slingshot"
-										src="/partners/slingshot.png"
-									/>
-									<Cell
-										alt="Versatile"
-										src="/partners/versatile.png"
-									/>
-									<Cell
-										alt="Versatile"
-										src="/partners/versatile.png"
-									/>
-									<Cell
-										alt="Versatile"
-										src="/partners/versatile.png"
-									/>
-								</SimpleGrid>
+									columns={[2, null, 3]}
+									spacing="20px"
+									minChildWidth={{
+										base: 170,
+										sm: 170,
+										md: 250,
+										lg: 300,
+									}}
+									spacingY={0}
+								></SimpleGrid>
 							</TabPanel>
 							<TabPanel>
 								<SimpleGrid
-									minChildWidth="250px"
+									columns={[2, null, 3]}
 									spacing="40px"
```

Comment:
```suggestion
									columns={{base: 2, md: null, lg: 3}}
									spacing={8}
```
Same as above (changed 40px to 20px for consistency's sake; if 40px is really needed, use {14}.)